### ISSUE TITLE
FillOrKill at off-market price results in PlaceOrder failure

### ISSUE THREAD

**Issue by GitteryCoder** [JKorf/Bitfinex.Net#40]
Hi JKorf,

When I call this function:
```
var placeResult = client.PlaceOrder(OrderType.ExchangeFillOrKill, "tBTCUSD", amt, clientOrderId:OrderID, price:(decimal)price);
```
I get success if my price results in a fill. But if I place an off-market price (say sell order at 6650 when the bid is 6600), the function waits ConfirmationTimeout and then returned as failed.

If I look at the logs though, it seems the order was placed and an "oc" was received quickly. Why does PlaceOrder consider it a failure then? Logs below.
```
2018/07/10 17:07:50:044 | Info | Going to place order
2018/07/10 17:07:50:072 | Debug | Sending [0,"on",null,{"type":"EXCHANGE FOK","symbol":"tBTCUSD","amount":"-0.002","price":"6650","cid":0}]
2018/07/10 17:07:50:398 | Debug | Received message: [0,"n",[1531213671928,"on-req",null,null,[14193804016,null,0,"tBTCUSD",null,null,-0.002,-0.002,"EXCHANGE FOK",null,null,null,null,null,null,null,6650,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null],null,"SUCCESS","Submitting exchange fok sell order for 0.002 BTC."]]
2018/07/10 17:07:50:398 | Debug | Received message: [0,"oc",[14193804016,null,0,"tBTCUSD",1531213671877,1531213671901,-0.002,-0.002,"EXCHANGE FOK",null,null,null,0,"FILLORKILL CANCELED",null,null,6650,0,null,null,null,null,null,0,0,0,null,null,"API>BFX",null,null,null]]
2018/07/10 17:07:50:398 | Debug | Processing [0,"n",[1531213671928,"on-req",null,null,[14193804016,null,0,"tBTCUSD",null,null,-0.002,-0.002,"EXCHANGE FOK",null,null,null,null,null,null,null,6650,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null],null,"SUCCESS","Submitting exchange fok sell order for 0.002 BTC."]]
2018/07/10 17:07:50:399 | Debug | Processing [0,"oc",[14193804016,null,0,"tBTCUSD",1531213671877,1531213671901,-0.002,-0.002,"EXCHANGE FOK",null,null,null,0,"FILLORKILL CANCELED",null,null,6650,0,null,null,null,null,null,0,0,0,null,null,"API>BFX",null,null,null]]
2018/07/10 17:07:59:877 | Debug | Received message: [0,"hb"]
2018/07/10 17:07:59:877 | Debug | Processing [0,"hb"]
2018/07/10 17:08:00:072 | Info | Failed to place order

```

**Comment by GitteryCoder** [comment]
Do I need to provide more information to help troubleshoot this issue? Am I the only one having problems with (slightly) off-markets FOKs returning "Failed to place order"?

**Comment by JKorf** [comment]
Hi, I've found the issue, it should be resolved in the latest version (v1.0.19)!

**Comment by GitteryCoder** [comment]
Brilliant! It works! Thank you very much!
